lib/handlers/compile.ts

```diff
@@ -441,9 +441,7 @@ export class CompileHandler implements ICompileHandler {
             backendOptions.skipPopArgs = query.skipPopArgs === 'true';
         }
         const executeParameters: ExecutionParams = {
-            args: Array.isArray(execReqParams.args)
-                ? execReqParams.args || ''
-                : utils.splitArguments(execReqParams.args),
+            args: execReqParams.args || [],
```

Comment:
Unfortunately I have told people that passing args as a string is OK here, please undo this